django/core/management/utils.py

```diff
@@ -75,3 +76,37 @@ def find_command(cmd, path=None, pathext=None):
             if os.path.isfile(fext):
                 return fext
     return None
+
+
+def parse_model_labels(labels, source=None):
```

Comment:
This function does two things:

1. Parse labels
2. Re-raise `LookupError`s as `CommandError`

I'd probably handle the latter step in `loaddata.py` and `dumpdata.py`. Also, I'm not sure whether changing output of a command is backward incompatible or not(In CPython it's not for example) . I don't think many people rely on exact output of Django management commands.